src/entities/Anubis.js

```diff
@@ -1,19 +1,18 @@
 import { GameObjects } from "phaser";
 import { Piece } from "./Piece";
-import { CELL_WIDTH, UNSELECTED_COLOR } from "../constants";
+import { CELL_HEIGHT, CELL_WIDTH } from "../constants";
 
-export class Djed extends Piece {
+export class Anubis extends Piece {
   constructor(params) {
     super(params);
 
-    this.graphic = new GameObjects.Star(
+    this.graphic = new GameObjects.Rectangle(
```

Comment:
Rectangle is reminiscent of an Obelisk (original name) and idk how to portray an Anubis with simple shapes. Next sprites...